File: Baekjoon/브루트포스/1977.py
# https://www.acmicpc.net/problem/1977

import sys
input = sys.stdin.readline

m = int(input())
n = int(input())

arr = []
for i in range(1, 101):
    if i ** 2 >= m and i **2  <= n:
        arr.append(i ** 2)
        
if len(arr) == 0:
    print(-1)
else:
    print(sum(arr))
    print(arr[0])

File: Baekjoon/브루트포스/1977.cpp
// https://www.acmicpc.net/problem/1977

#include <iostream>
#include <cmath>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int m, n;
    cin >> m >> n;
    int sum = 0, min = 10001;
    for (int i = m; i <= n; i++) {
        if (sqrt(i) == (int)sqrt(i)) {
            sum += i;
            min = i < min ? i : min;
        }
    }
    if (sum == 0) cout << -1 << '\n';
    else cout << sum << '\n' << min << '\n';
    return 0;
}